tests/acceptance/features/apiSpacesShares/shareSpacesViaLink.feature

```diff
@@ -128,3 +128,65 @@ Feature: Share spaces via link
     Then the HTTP status code should be "400"
     And the OCS status message should be "Can not share space root"
     And for user "Alice" the space "Alice Hansen" should not contain the last created public link
+
+  Scenario: remove password of a public link space share with read permission as a Space Admin
+    Given user "Alice" has created a public link share of the space "share space" with settings:
+      | permissions | 1        |
+      | password    | %public% |
+    When user "Alice" updates the last public link share using the sharing API with
+      | permissions | 1 |
+      | password    |   |
+    Then the HTTP status code should be "200"
+    And the OCS status code should be "100"
+
+
+  Scenario Outline: remove password of a public link space share with read permission as a Space Admin
```

Comment:
Is it read permission for this scenario and for this scenario in L175?

Reply:
No, it's not. I copied and pasted and forgot to update I think. Thanks, I'll update them!